Sources/ProjectSpec/TargetSource.swift

```diff
@@ -192,6 +195,7 @@ extension TargetSource: JSONObjectConvertible {
 
         createIntermediateGroups = jsonDictionary.json(atKeyPath: "createIntermediateGroups")
         attributes = jsonDictionary.json(atKeyPath: "attributes") ?? []
+        resourceTags = jsonDictionary.json(atKeyPath: "resourceTags") ?? []
```

Comment:
You'll need to add a line in `toJSONValue` down below as well

Reply:
e038d9b